src/components/Calculator.js

```diff
@@ -24,7 +25,9 @@ const Calculator = ({ children }) => {
 
   return (
     <div className="Wrapper">
+      <Quote />
```

Comment:
- [ ]  Great work on importing your Quote component! Just a quick heads-up: according to the project requirements, could you please place it next to your Calculator component in the main file (App.js)?

   ![image](https://github.com/MohammadYaser/math-magicians/assets/107952724/27fecc9a-8e20-4419-8d6f-50d00aee78e5)